.shellcheckrc

```diff
@@ -0,0 +1 @@
+disable=SC3028,SC3054,SC3014,SC3040,SC3046,SC3030,SC3010,SC3037,SC3045,SC3006,SC3018,SC3016,SC3011,SC3044,SC3043,SC3060,SC3024
```

Comment:
Uhm, I prefer to resolve all these errors, but I'm ok with working on those resolving in a separate PR.
@Vandit1604 WDYT?

Appendix: https://www.shellcheck.net/wiki/

Reply:
Sure, but I would like to solve them in separate PR(s).

Reply:
I'm ok with a separate PR. Could you open an issue to keep tracking this improvement?

Reply:
Filed https://github.com/kubernetes-sigs/kueue/issues/1795 for tracking the progress.

@tenzen-y

Reply:
Thans